rule.go

```diff
@@ -914,3 +911,14 @@ func (r *Rule) HasVar(s string) bool {
 	}
 	return false
 }
+
+//getSidMgs returns a string representing a sidmsg.map entry
+func (r *Rule) getSidMgs(s string) string {	
```

Comment:
There are no tests for this function. Please add some tests to `rule_test.go` that cover several cases (no reference, one reference, many references, etc.)

Reply:
3 tests have been added, with and without References.